server/data/activeLduClient.js

```diff
@@ -0,0 +1,23 @@
+const db = require('./dataAccess/db')
+
+module.exports = {
+  async addLdu(lduCode) {
+    const query = {
+      text: `INSERT INTO active_local_delivery_units (ldu_code) VALUES ($1) ON CONFLICT (ldu_code) DO UPDATE SET ldu_code=active_local_delivery_units.ldu_code RETURNING ldu_code`,
+      values: [lduCode],
```

Comment:
probably don't want to do an update on conflict, would be tempted to `ON CONFLICT DO NOTHING`.

Reply:
I did consider doing ON CONFLICT DO NOTHING but then came across this article that says that nothing will be returned on a conflict. https://intellipaat.com/community/14280/how-to-use-returning-with-on-conflict-in-postgresql

So I used DO UPDATE just so we have a complete object on return